R/utils.R

```diff
@@ -226,9 +226,9 @@ tf_formats <- function() {
 
   dplyr::tribble(
```

Comment:
Before, it seemed that `tf_formats_icons()` included NAs, not sure what the impact of all this is tbh.

Now, it is:

```
tf_formats_icons()
[1] "c(\"✔\", \"✘\")" "c(\"●\", \"⭘\")" "c(\"■\", \"□\")" "c(\"◆\", \"◇\")" "c(\"↑\", \"↓\")" "c(\"▲\", \"▼\")"
[7] "c(\"▶\", \"◀\")"
```

which seems to make sense?

Reply:
Thanks for adding `omit_na()`! Yeah, this return value is correct now but AFAICT nothing in the package uses `tf_formats_icons()` (so I guess no impact).